test/unit-tests/MultiBridgeMessageSender.t.sol

```diff
@@ -492,10 +492,10 @@ contract MultiBridgeMessageSenderTest is Setup {
 
         sender.addSenderAdapters(adapters);
 
-        assertEq(sender.senderAdapters(0), wormholeAdapterAddr);
-        assertEq(sender.senderAdapters(1), axelarAdapterAddr);
-        assertEq(sender.senderAdapters(2), higherAddr0);
-        assertEq(sender.senderAdapters(3), higherAddr1);
+        assertEq(sender.senderAdapters(0), axelarAdapterAddr);
```

Comment:
This test is meant to cover the case where added adapters have higher address than all existing ones, so please change `higherAddr0` to `wormholeAdapterAddr + 1` to keep the coverage.

Reply:
Done